crfs.go

```diff
@@ -1009,6 +1009,46 @@ func (n *node) Readlink(ctx context.Context, req *fuse.ReadlinkRequest) (string,
 	return n.te.LinkName, nil
 }
 
+// Listxattr lists the extended attributes specified for the node.
+//
+// See https://godoc.org/bazil.org/fuse/fs#NodeListxattrer
+func (n *node) Listxattr(ctx context.Context, req *fuse.ListxattrRequest, resp *fuse.ListxattrResponse) error {
+	var allXattrs []byte
+	for k, _ := range n.te.Xattrs {
+		allXattrs = append(append(allXattrs, []byte(k)...), 0)
+	}
+
+	if req.Position >= uint32(len(allXattrs)) {
+		resp.Xattr = []byte{}
+		return nil
+	}
+	allXattrs = allXattrs[req.Position:]
+	if req.Size > 0 && req.Size < uint32(len(allXattrs)) {
+		return syscall.ERANGE
+	}
+	resp.Xattr = allXattrs
+	return nil
+}
+
+// Getxattr reads the specified extended attribute.
+//
+// See https://godoc.org/bazil.org/fuse/fs#NodeGetxattrer
+func (n *node) Getxattr(ctx context.Context, req *fuse.GetxattrRequest, resp *fuse.GetxattrResponse) error {
+
+	xattr := n.te.Xattrs[req.Name]
+	if req.Position >= uint32(len(xattr)) {
+		resp.Xattr = []byte{}
+		return nil
+	}
+	xattr = xattr[req.Position:]
+	if req.Size > 0 && req.Size < uint32(len(xattr)) {
+		return syscall.ERANGE
```

Comment:
Same as above, returning syscall.ERANGE results in EIO.
Simply, by deleting this check all should work fine because bazil/fuse do this check in a proper way